### ISSUE TITLE
new lines in args 

### ISSUE THREAD

**Issue by dvyukov** [google/trace-viewer#751]
Hi,

I generate events like this:

{"name":"syscall","ph":"I","ts":11873,"pid":0,"tid":534,"s":"t","args":{"stack":"runtime.reentersyscall\nruntime.entersyscall\nsyscall.Syscall\nsyscall.read\nsyscall.Read\nnet.(*netFD).Read\nnet.(*conn).Read\nnet/http.(*liveSwitchReader).Read\nio.(*LimitedReader).Read\nbufio.(*Reader).fill\nbufio.(*Reader).ReadSlice\nbufio.(*Reader).ReadLine\nnet/textproto.(*Reader).readLineSlice\nnet/textproto.(*Reader).ReadLine\nnet/http.ReadRequest\nnet/http.(*conn).readRequest\nnet/http.(*conn).serve\n"}},

and want \n to be replaced with actual new lines in browser.

It works with trace2html. But when I use vulcanize, \n are replaced with spaces.
I am on branch releases_chrome_m41.

Please fix vulcanize so that it works as trace2html.

**Comment by natduca** [comment]
This should be a pretty easy patch to trace2html's python code. Wanna try doing a pull-request?

**Comment by natduca** [comment]
Actually I dont understand what is broken. I just tried pointing bin/index.html at a trace with newlines in it and it worked. So I need a repro in order to do anything with this bug.

**Comment by dvyukov** [comment]
This is no longer relevant after #752
